pom.xml

```diff
@@ -64,6 +79,19 @@
             <url>http://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
+    <dependencies>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>matrix-project</artifactId>
+            <version>1.0</version>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-step-api</artifactId>
+            <version>1.4</version>
```

Comment:
:ant: `workflow-step-api` was expressly designed to avoid the need for `optional` dependencies: it contributes no UI or other runtime behavior on its own, it is just API.